ocaml/xapi/xapi_pool.ml

```diff
@@ -1620,6 +1620,20 @@ let eject ~__context ~host =
       (fun () -> Xapi_fuse.light_fuse_and_reboot_after_eject ()) ;
     Xapi_hooks.pool_eject_hook ~__context
 
+(** eject [host] from the pool. This code is run on all hosts in the
+pool but only [host] will eject itself. *)
+let eject ~__context ~host =
```

Comment:
This is surprising, can the existing eject in line 1387 have its name changed? Maybe `self_eject`?